keyboards/handwired/tritium_numpad/rules.mk

```diff
@@ -0,0 +1,63 @@
+
+# MCU name
+#MCU = at90usb1287
+MCU = atmega32u4
+
+# Processor frequency.
+#     This will define a symbol, F_CPU, in all source code files equal to the
+#     processor frequency in Hz. You can then use this symbol in your source code to
+#     calculate timings. Do NOT tack on a 'UL' at the end, this will be done
+#     automatically to create a 32-bit value in your source code.
+#
+#     This will be an integer division of F_USB below, as it is sourced by
+#     F_USB after it has run through any CPU prescalers. Note that this value
+#     does not *change* the processor frequency - it should merely be updated to
+#     reflect the processor speed set externally so that the code can use accurate
+#     software delays.
+F_CPU = 16000000
+
+#
+# LUFA specific
+#
+# Target architecture (see library "Board Types" documentation).
+ARCH = AVR8
+
+# Input clock frequency.
+#     This will define a symbol, F_USB, in all source code files equal to the
+#     input clock frequency (before any prescaling is performed) in Hz. This value may
+#     differ from F_CPU if prescaling is used on the latter, and is required as the
+#     raw input clock is fed directly to the PLL sections of the AVR for high speed
+#     clock generation for the USB and other AVR subsections. Do NOT tack on a 'UL'
+#     at the end, this will be done automatically to create a 32-bit value in your
+#     source code.
+#
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Bootloader
+#     This definition is optional, and if your keyboard supports multiple bootloaders of
+#     different sizes, comment this out, and the correct address will be loaded
+#     automatically (+60). See bootloader.mk for all options.
+BOOTLOADER = caterina
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration(+1000)
```

Comment:
Does this have a physical reset button? 

If not, you may want to set this to `full` or `lite`, instead.

Reply:
The promicro has a wire jumper (you stick it between the reset and ground pins) that makes it go into the bootloader.  There isn't a dip switch.

Reply:
The "Virtual DIP switch" comment is a bit misleading, Bootmagic doesn't really relate to DIP switches.

Reply:
What fauxpark said. 

Specifically, is there a button of any kind connected to the RST and GND pins?   

Or some other way to get into bootloader without having to manually short the pins? 

If not, then Bootmagic has the ability to reset to bootloader by holding specific keys when plugging in the board (well when the firmware starts).

Reply:
Thanks for your patience.  Although the promicro is exposed to the outside world (no cover), it would be unreasonable to expect somebody to have a wire jumper sitting around.

Does it make sense to set the numlock key as a bootmagic lite key?

Reply:
Sounds like a good idea. Generally the Bootmagic Lite key is the top left switch as that is usually (0,0) in the matrix.

Reply:
You can find the config stuff for lite here:
https://docs.qmk.fm/#/feature_bootmagic?id=bootmagic-lite

Reply:
I was able to test the numpad with the bootmagic (lite).  I'll commit the changes.